packages/safe-ds-eda/src/App.svelte

```diff
@@ -0,0 +1,87 @@
+<script lang="ts">
+    import Debugger from './components/Debugger.svelte';
+    import TableView from './components/TableView.svelte';
+    import Sidebar from './components/Sidebar.svelte';
+    import { throttle } from 'lodash';
+
+    let sidebarWidth = 307; // Initial width of the sidebar in pixels
+
+    // Function to handle the dragging
+    function handleDrag(e: MouseEvent) {
+        const throttledOnMouseMove = throttle(onMouseMove, 30);
+
+        let prevX = e.clientX;
+
+        function onMouseMove(e: MouseEvent) {
+            const dx = e.clientX - prevX;
+            prevX = e.clientX;
+            if (sidebarWidth + dx < 10) {
+                sidebarWidth = 10;
+            } else {
+                sidebarWidth += dx;
+            }
+        }
+
+        function onMouseUp() {
+            document.removeEventListener('mousemove', throttledOnMouseMove);
+            document.removeEventListener('mouseup', onMouseUp);
+            document.removeEventListener('mousemove', clearTextSelection);
+        }
+
+        document.addEventListener('mousemove', throttledOnMouseMove);
+        document.addEventListener('mouseup', onMouseUp);
+        document.addEventListener('mousemove', clearTextSelection);
+    }
+
+    function clearTextSelection() {
+        const selection = window.getSelection();
+        if (selection) selection.removeAllRanges();
+    }
+</script>
+
+<main>
+    <!-- <Debugger /> -->
```

Comment:
Is this a feature we need for the release or just a development help? In the latter case, remove it please.